docs/destinations/warehouses-and-lakes/rdb/previous-versions/rdb-loader-1-1-0/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "RDB Loader 1.1.0"
 date: "2021-06-08"
-sidebar_position: -10
+sidebar_position: 0
```

Comment:
Could you order the previous versions in descending order, like for other components?

Reply:
Sure thing. On it 🏃‍♀️